account_invoice_prices_update/wizards/account_invoice_prices_update_wizard.py

```diff
@@ -74,6 +74,7 @@ def update_prices(self):
             line._onchange_balance()
             line._onchange_mark_recompute_taxes()
             line.move_id._onchange_invoice_line_ids()
+            line.move_id.message_post(body='The pricelist is now: %s' % self.pricelist_id.display_name)
```

Comment:
If you do it that way, it will happen that the message will be written "n" times, where "n" is the number of lines with product that you have on the invoice.

Is better doing like this:`invoice.message_post(body='The pricelist is now: %s' % self.pricelist_id.display_name)`